cmd/dfdaemon/app/version.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"fmt"
+
+	"github.com/dragonflyoss/Dragonfly/version"
+
+	"github.com/spf13/cobra"
+)
+
+var versionCmd = &cobra.Command{
+	Use:           "version",
+	Short:         "Show the current version of dfdaemon",
```

Comment:
Could you add a `Long` field for every command, like 
```
Long:              "The dfdaemon is a proxy between container engine and registry used for pulling images.",
```

In addition, add an example for every command, like https://github.com/alibaba/pouch/blob/master/cli/create.go#L36.
It could be helpful when user executed `command -h`.

Reply:
Done